docs/04-Captures/cli.md

```diff
@@ -1,203 +1,279 @@
 # Capture with Retina CLI
 
-Retina capture command allows the user to capture network traffic and metadata for the capture target, and then send the capture file to the location by Output Configuration.
+The Retina capture command allows users to capture network traffic and metadata for the capture target, and send the data to the location defined by output configuration.
 
-Note: captures can also be performed with a [Capture CRD](../05-Concepts/CRDs/Capture.md) after [installing Retina](../02-Installation/01-Setup.md) with capture support.
+> Note: captures can also be performed with a [Capture CRD](../05-Concepts/CRDs/Capture.md) after [installing Retina](../02-Installation/01-Setup.md) **with capture support**.
 
-## Retina capture create
+## Starting a Capture
 
-`kubectl retina capture create` creates a Capture with underlying Kubernetes jobs.
+`kubectl retina capture create [--flags]` creates a Capture with underlying Kubernetes jobs.
 
-### No wait(default true)
+## Flags
 
-Do not wait for the long-running capture job to finish.
+### [required] Name
 
-By default, Retina capture CLI will exit before the jobs are completed. With `--no-wait=false`, the CLI will wait until the jobs are completed and clean up the Kubernetes resources created.
+- Syntax: `--name <string>`
```

Comment:
It's a cli command so the syntax is more or less assumed, what I would do here instead could be a tabulated form something like

|flags|description|
|---|---|
|--name|[Required] Some Desc |
|-x --xero| Some Desc |

Reply:
I suppose the syntax bit is a bit overkill since we already have examples for each flag showcasing usage. On the other hand is there value in deleting it? I don't have a super strong opinion either way.

In terms of the table, I like the idea, I'm just a bit worried about how it would render if we want to have all the information in it. Especially since some of the descriptions / examples are quite long.

| Flag | Required | Default Value | Description | Example |
|--------|--------|--------|--------|--------|
| --name | yes |  | Description | `kubectl retina capture create --name capture-test --host-path /mnt/capture --node-selectors "kubernetes.io/os=linux"`|

Reply:
I would just keep the first 4 columns and move examples to it's own subsection, see the below comment, maybe that would make my point clearer.

Reply:
I will keep it as is for now just to not block this PR and other changes - but I will look into addressing the examples in the next PR.